spec/integration/outputs/update_spec.rb

```diff
@@ -53,8 +53,19 @@ def get_es_output( options={} )
       r = @es.get(:index => 'logstash-update', :type => 'logs', :id => "123", :refresh => true)
       insist { r["_source"]["message"] } == 'updated message here'
     end
+
+    # The es ruby client treats the data field differently. Make sure this doesn't
+    # raise an exception
+    it "should update an existing document that has a 'data' field" do
+      subject = get_es_output({ 'document_id' => "123" })
+      subject.register
+      subject.receive(LogStash::Event.new("data" => "updated message here"))
```

Comment:
can we make this event include a sibling field beside `"data"`, just to be more explicit that not all your fields necessarily will be found in `data`